test/acceptance/features/bindAppToService.feature

```diff
@@ -531,3 +531,57 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
         And Application endpoint "/api/todos" is available
+
+    Scenario: Bind values from service CRD annotation
```

Comment:
Can we/ should we change `bind` to `export` ?

Reply:
I just followed the previous test scenarios in this file. 
`Bind an imported Node.js application to Etcd database`
`Bind an imported Node.js application to PostgreSQL database in the following order: Service Binding, DB and Application`

Reply:
I am going for this: `Provide binding info through backing service CRD annotation`